roles/edpm_update/molecule/default/molecule.yml

```diff
@@ -0,0 +1,38 @@
+---
+dependency:
+  name: galaxy
+  options:
+    role-file: collections.yml
+driver:
+  name: podman
+platforms:
+- command: /sbin/init
+  dockerfile: ../../../../molecule/common/Containerfile.j2
```

Comment:
Sorry to be picking on this PR again, but I am using this PR to build my change on top of it.
The [containerfile](https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/molecule/common/Containerfile.j2#L9) will run `dnf upgrade -y` so essentailly the DUT(instance) is already updated before the converge of this happens. So in the end, testing of this update role is not testing much. Shouldnt the prepare state of this setup some DUT which could be updated and then converge edpm_update role and verify?

Reply:
You're right, i guess that we can make an exception for this role.